server/container/guice/cassandra-guice/src/main/java/org/apache/james/modules/mailbox/CassandraBlobStoreModule.java

```diff
@@ -36,12 +40,17 @@
     protected void configure() {
         bind(CassandraDefaultBucketDAO.class).in(Scopes.SINGLETON);
         bind(CassandraBlobStore.class).in(Scopes.SINGLETON);
+        bind(CassandraBlobExistenceTesterDAO.class).in(Scopes.SINGLETON);
+        bind(CassandraBlobExistenceTester.class).in(Scopes.SINGLETON);
```

Comment:
Why do we need this class specific for object storage in a oroject where we dont have an object storage blob store at hand?